core/templates/dev/head/services/explorationContextService.js

```diff
@@ -20,7 +20,8 @@
 oppia.constant('PAGE_CONTEXT', {
   EDITOR: 'editor',
   LEARNER: 'learner',
-  OTHER: 'other'
+  OTHER: 'other',
+  EMBED: 'iframe'
```

Comment:
Why do you need to make all these changes? I'm not sure they're used anywhere, and if that's the case I recommend reverting them.